src/pages/signUp/SignUpPage.tsx

```diff
@@ -1,43 +1,54 @@
-import { Main, MainHeadline } from "../../ui";
-import React, { ChangeEvent, FormEvent, RefObject, useState } from "react";
-import { Button, Box, TextInput } from "../../ui";
+import {
+  Box,
+  Button,
+  Main,
+  MainHeadline,
+  TextInput,
+  ValidationError,
+} from "../../ui";
+import React, { useCallback, useState } from "react";
 import cx from "classnames";
 import { useAuthentication } from "../../features/authentication";
+import Form, { Collection, useValidation } from "usetheform";
+import {
+  validateEmail,
+  validateRequiredEmail,
+  validateRequiredPasswords,
+  validatePasswords,
+  PasswordState,
+} from "../utils/validation";
+
+interface FormState {
+  email: string;
+  passwords: PasswordState;
+}
+
+const initialFormState: FormState = {
+  email: "",
+  passwords: {
+    password: "",
+    password2: "",
+  },
+};
 
 export default function SignUpPage() {
+  const [, setFormState] = useState(initialFormState);
```

Comment:
It seems that you are not using the state:

const [, setFormState] = useState(initialFormState);